srtcore/api.cpp

```diff
@@ -1504,22 +1504,22 @@ void CUDTUnited::checkBrokenSockets()
       j != m_ClosedSockets.end(); ++ j)
    {
       // HLOGF(mglog.Debug, "checking CLOSED socket: %d\n", j->first);
-      if (j->second->m_pUDT->m_ullLingerExpiration > 0)
+      if (j->second->m_pUDT->m_tsLingerExpiration_us.value != 0)
```

Comment:
Right.

Reply:
Again; I'm worried whether the compiler will optimize it into a simple integer number comparison, or will bounce around wrappers. This kind of comparison will be better from the readability point of view, but it will actually request a temporary object of `ClockSys` initialized to 0 and then use the user-defined != operator. I'm not sure if the optimized code will really result with the same simple integer comparison as above.